# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../perl/typo_hotfix.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From bddecfb07990891aab7da507187980d9333ee0af Mon Sep 17 00:00:00 2001
From: Alex Suykov <alex.suykov@gmail.com>
Date: Wed, 23 Apr 2014 10:10:34 +0300
Subject: [PATCH] typo

---
 cnf/configure_libs.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cnf/configure_libs.sh b/cnf/configure_libs.sh
index f45d31c..e970947 100644
--- a/cnf/configure_libs.sh
+++ b/cnf/configure_libs.sh
@@ -43,9 +43,9 @@ if not hinted 'perllibs'; then
 	# should be linked with.
 	# The whole idea is wrong, wrong, wrong, but it's tied to MakeMaker.
 	# Unlike Configure, we're picking libs presumably needed for perl
-	# (Configure uses all except for those it know are not needed)
-	# This allows adding anything to $libswanted without introducing unnecessary perl
-	# dependencies. When perl itself needs something unusual, $perllibs value should be hinted.
+	# (Configure uses all except for those it knows are not needed)
+	# This allows adding anything to $libswanted without introducing unnecessary perl dependencies.
+	# When perl itself needs something unusual, $perllibs value should be hinted.
 	_libs=''
 	for i in $libs; do
 		case "$i" in 
-- 
1.9.3

